HelloMixpanel/HelloMixpanelTests/ABTestingTests.m

```diff
@@ -91,6 +91,21 @@ - (void)stubDecide:(NSString *)path {
     .withBody([NSData dataWithContentsOfURL:responseURL]);
 }
 
+- (void)waitForABTest {
```

Comment:
we need to wait for main queue now because we moved variant execution onto the main queue